docs/examples/kops-test-route53-subdomain.md

```diff
@@ -0,0 +1,870 @@
+# USING KOPS WITH A ROUTE53 BASED SUBDOMAIN AND SCALING UP THE CLUSTER
+
+## WHAT WE WANT TO ACOMPLISH HERE ?.
+
+The exercise described on this document will focus on the following goals:
+
+- Demonstrate how to use a production-setup with 3 masters and two workers in different availability zones.
+- Ensure our masters are deployed on 3 different AWS availability zones.
+- Ensure our nodes are deployed on 2 different AWS availability zones.
+- Use AWS Route53 service for the cluster DNS sub-domain.
+- Show how to properly scale-up our cluster.
+
+
+## PRE-FLIGHT CHECK:
+
+Before rushing in to replicate this exercise, please ensure your basic environment is correctly setup. See the [KOPS AWS tutorial for more information](https://github.com/kubernetes/kops/blob/master/docs/aws.md). 
+
+Ensure that the following points are covered and working in your environment:
+
+- "jq" utility installed (this is available on most linux distributions). If you are running on Centos, you'll need to add "epel" repository with `yum -y install epel-release` then install jq with `yum -y install jq`.
+- "dig" utility installed (this is also available on most linux distributions). We'll need "dig" in order to tests our DNS subdomain. On "centos/rhel" distros, this utility is part of the "bind-utils" package.
+- AWS cli fully configured (aws account already with proper permissions/roles needed for kops). Depending on your distro, you can setup directly from packages, or if you want the most updated version, use "pip" and install awscli by issuing a "pip install awscli" command. Your choice !.
+- Local ssh key ready on ~/.ssh/id_rsa / id_rsa.pub. You can generate it using "ssh-keygen" command: `ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ""`
+- Region set to us-east-1 (az's: us-east-1a, us-east-1b, us-east-1c, us-east-1d and us-east-1e). For this exercise we'll deploy our cluster on US-EAST-1. For real HA at kubernetes master level, you need 3 masters. If you want to ensure that each master is deployed on a different availability zone, then a region with "at least" 3 availabity zones is required here. You can still deploy a multi-master kubenetes setup on regions with just 2 az's, but this mean that two masters will be deployed on a single az, and of this az goes offline then you'll lose two master !. If possible, always pick a region with at least 3 different availability zones for real H.A. You always can check amazon regions and az's on the link: [AWS Global Infrastructure](https://aws.amazon.com/about-aws/global-infrastructure/)
```

Comment:
In us-east1? You have "deploy our cluster on"

Reply:
If you have 2 az I would recommend deploying all masters to a single az. What are your thoughts?

Reply:
You have a !. Again ... just!

Also can we mention that we are talking about the masters ha not the cluster ha? 3 masters provide control plane HA.